cicd/builds/triggering-builds-build-hooks.adoc

```diff
@@ -30,8 +30,12 @@ include::modules/builds-configuration-change-triggers.adoc[leveloffset=+2]
 
 include::modules/builds-setting-triggers-manually.adoc[leveloffset=+3]
 
+include::modules/unauthenticated-users-system-webhook.adoc[leveloffset=+2]
```

Comment:
This is now a lot better.

Would it make sense to put this before we give examples for how to setup GitHub and GitLab and so forth, such that the user doesn't wonder why it doesn't work? If the user reads that before reading the GitHub setup, the users knows that there is such a thing.

We should also mention that this is a prerequisite for it to work.